wikipedia-dark.user.css

```diff
@@ -419,6 +419,39 @@
 @-moz-document regexp("https?:\/\/(([\w\-]{2,}\.)?(wiki(pedia|books|news|quote|source|versity|voyage)|wiktionary)|(www|test)\.(mediawiki|wiki(data|dot))|(meta|wikimania|commons|otrs-wiki|species|wikitech|incubator|outreach|foundation|donate)\.wikimedia)\.org\/.*"),
                regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$"),
                regexp("https?:\/\/wikiless\.org\/.*$") {
+  #vector-toc {
+    background-color: transparent;
+  }
+  .tracklist th[scope="col"] {
+    background-color: var(--gray-3) !important;
+  }
+  caption {
+    background-color: var(--gray-3) !important;
+    border-color: var(--gray-5) !important;
+  }
+  button.mw-collapsible-toggle {
+    border-color: var(--gray-5);
+  }
+  button.mw-collapsible-toggle::after,
+  button.mw-collapsible-toggle::before {
+    color: var(--gray-5);
+  }
+  .tracklist-total-length span,
+  .tracklist-total-length td {
+    background-color: var(--gray-3) !important;
+  }
+  .infobox-data {
+    background-color: var(--gray-2) !important;
+  }
+  .infobox-header {
+    border-color: var(--gray-5) !important;
+  }
```

Comment:
These two rules no longer seem to be necessary after #206 was applied. If you find an area where it appears to be needed, we can revisit this one. Also try to avoid low specificity selectors using `!important`, where possible.